### ISSUE TITLE
Count alleles currently breaks with multiple column chunks

### ISSUE THREAD

**Issue by eric-czech** [pystatgen/sgkit#79]
The results from it are correct but this assert statement:

https://github.com/pystatgen/sgkit/blob/21a03c934d202e6c6e0939d43df78bc1f3ad0092/sgkit/stats/aggregation.py#L60

should be:

```python
assert CT.shape == (n_variant, G.numblocks[1] * max_allele)
```

**Comment by tomwhite** [comment]
Good catch. Do you have a case that breaks? It would be good to add as a test.

**Comment by tomwhite** [comment]
Fixed in #93
